test/lish/archives_and_uploads_a_package_test.dart

```diff
@@ -15,10 +15,9 @@ import '../test_pub.dart';
 import 'utils.dart';
 
 void main() {
-  setUp(d.validPackage.create);
```

Comment:
Just curious, why?

Reply:
I find self-contained tests much easier to read.